### DISCUSSION TITLE
SyntaxError: Cannot use import statement outside a module

### DISCUSSION THREAD

**Discussion by namrataajmeri16** [axios/axios#6181] [Q&A] [answered]
We are using node v16.16.0 and i tried to update axios from 0.24.0 to 1.6.3. It successfully gets updated but the jest test cases are failing.

.../node_modules/axios/index.js:1
    ({"Object.<anonymous>":function(module,exports,require,__dirname,__filename,global,jest){import axios from './lib/axios.js';

    SyntaxError: Cannot use import statement outside a module

  const mockAxios = jest.genMockFromModule('axios'); 

in package.json we haven't mentioned "type": "module"

Could anyone please look into this issue? This update is failing lot of our jest test cases.

**Comment by leandrobravo** [comment]
@namrataajmeri16 have you solved this issue?

**Comment by mchenx** [comment, marked as answer]
@namrataajmeri16 @leandrobravo 
you need to upgrtade jest to 29.2.x or later, pls refer to this #5101
